sentence_transformers/models/Transformer.py

```diff
@@ -156,8 +156,8 @@ def tokenize(self, texts: Union[List[str], List[Dict], List[Tuple[str, str]]]):
     def get_config_dict(self):
         return {key: self.__dict__[key] for key in self.config_keys}
 
-    def save(self, output_path: str):
-        self.auto_model.save_pretrained(output_path)
+    def save(self, output_path: str, safe_serialization: bool):
```

Comment:
```suggestion
    def save(self, output_path: str, safe_serialization: bool = True):
```
This prevents people from suddenly experiencing a crash if they some reason call this method directly with just `save("my/path")`.

Reply:
Yeah, you're right.